modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -28,15 +28,24 @@ class AlephClient {
 
   /**
    * The base URL for the REST service.
+   *
    * @var string
    */
   protected $baseUrlRest;
 
   /**
-   * The primary library, ICE01 for example.
+   * The catalog library, ICE01 for example.
+   *
+   * @var string
+   */
+  protected $catalogLibrary;
+
+  /**
+   * The item library, ICE53 for example.
```

Comment:
:+1: for the addition here. If would be nice if we could provide any form of documentation regarding what we mean by catalog and item library.